reporters/reporter-perfrepo/src/main/java/org/radargun/reporting/perfrepo/PerfrepoReporter.java

```diff
@@ -194,10 +208,18 @@ private void addConfigsAndUpload(TestExecutionBuilder testExecutionBuilder, Repo
       addNormalizedConfigs(report, testExecutionBuilder);
       // create new test execution
       TestExecution testExecution = testExecutionBuilder.build();
+      if (testExecution.getParameters() != null) {
+         for (TestExecutionParameter parameter : testExecution.getParameters()) {
+            String value = parameter.getValue();
+            if (value != null && value.length() > 2047) {
+               parameter.setValue(value.substring(0, 2044) + "...");
```

Comment:
Would be good an explanation about those magic numbers

Reply:
See the property `value` in `TestExecutionParameter`

Reply:
> See the property `value` in `TestExecutionParameter`
Ok, maybe a const with a describable name instead of the literal value?

Reply:
Done